app/i18n/languages/en.js

```diff
@@ -157,5 +157,6 @@ export default {
 	'Delete account model error': 'It was not possible to delete your account because you have models. Please remove all your models before deleting your account.',
 	'Forbidden access': 'Forbidden access',
 	'Looks like this model does not exist or you don\'t have access to it.': 'Looks like this model does not exist or you don\'t have access to it.',
-	'Back to models list': 'Back to models list'
-};
\ No newline at end of file
+	'Back to models list': 'Back to models list',
+	'Copy and paste is not allowed on this module due to complexity when updating references': 'Copy and paste is not allowed on this module due to complexity when updating references'
```

Comment:
Is this copy being used?

Reply:
Good catch! It was my first version but found too long, changed and forgot this here.

The new one will be:

'Copy is not allowed on this module when element has references.'
'Copiar não é permitido nesse módulo quando elementos já estiverem conectados.'

What do you think?

Reply:
Looks good to me. Will you update it here or in a new branch?

Reply:
Here, I am working on it right now. Let's try merge this today.